src/tests/ftest/util/soak_utils.py

```diff
@@ -233,7 +254,7 @@ def get_daos_server_logs(self):
         self (obj): soak obj
     """
     daos_dir = self.outputsoak_dir + "/daos_server_logs"
-    logs_dir = os.environ.get("DAOS_TEST_LOG_DIR", "/var/tmp/daos_testing/")
+    logs_dir = os.environ.get("DAOS_TEST_LOG_DIR", "/tmp/")
```

Comment:
This should always be defined now
```suggestion
    logs_dir = self.test_env.log_dir
```